FCCee/CLD/compact/README.md

```diff
@@ -26,6 +26,13 @@ This model was taken from FCCDetectors `FCCee_o2_v03`. It presents an alternate
 with a smaller radius, which preserves angular coverage of the vertex barrel. Various issues of the Tracker structure
 where inherited from the FCCDetectors `FCCee_o2_v02` model.
 
+CLD_o2_v05
+----------
+
+This model is based in `FCCee_o2_v02` from k4geo. It has an updated design of the beampipe corresponding to the latest
```

Comment:
```suggestion
This model is based on `FCCee_o2_v02` from k4geo. It has an updated design of the beampipe corresponding to the latest
```

Reply:
Corrected. And added that the HOM Absorber is not needed any more.